### PR TITLE
ocko_1997_halide_55

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    
+curator:
+    version: 1
+    name: Markus Schilling
+    orcid: https://orcid.org/0000-0003-2684-3711
+    digitized: 2021-10-18 # YYYY-MM-DD
+    
+figure description: #This section will be updated by module `CV`
+    version: 1
+    linked measurements: Synchrotron X-ray scattering # FTIR, SXRD, Ring Disc - indicate if any other operando methods were shown in the same plot, which is linked to the data
```
Comment (DunklesArchipel) [inline literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml:20]: SXRD ?

**Review by DunklesArchipel — APPROVED** [review]
Maybe change the acronym for the technique.

File: literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml (hunk 1 above)
Comment (markus-chem) [reply literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml:20]: For me, SXRD usually refers to single crystal XRD, but can be also Surface XRD. Synchrotron XRD as SXRD would be new for me. But I am fine with all options.

Comment (DunklesArchipel) [reply literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml:20]: SXRD refers to "surface  X-ray diffraction" which is often conducted on single crystal electrodes. So the "S" does not stand for "synchrotron".

Comment (markus-chem) [reply literature/ocko_1997_halide_55/ocko_1997_halide_55_p6_6a.yaml]: Changed to Synchrotron XRD

**Pull request merged**